package/MDAnalysis/coordinates/PDB.py

```diff
@@ -56,44 +56,24 @@
 Implementations
 ---------------
 
-Two different implementations of PDB I/O are available: the
-":ref:`permissive<permissive>`" and the ":ref:`strict<strict>`" Reader/Writers.
-The default are the "permissive" ones but this can be changed by setting the
-flag "permissive_pdb_reader" in :data:`MDAnalysis.core.flags` (see
-:ref:`flags-label`) to ``False``::
+PDB I/O is available in the form of the":ref:`permissive<permissive>`"
```

Comment:
I think you can remove the mention of permissive here since we only have one PDB module left.